juriscraper/opinions/united_states/state/tex.py

```diff
@@ -139,3 +168,51 @@ def get_name(link: str) -> Optional[str]:
                 return None
 
         return DeferringList(seed=seeds, fetcher=get_name)
+
+    def make_backscrape_iterable(self, kwargs: Dict) -> None:
+        """Checks if backscrape start and end arguments have been passed
+        by caller, and parses them accordinly
+
+        :return None
+        """
+        start = kwargs.get("backscrape_start")
+        end = kwargs.get("backscrape_end")
+
+        if start:
+            start = datetime.strptime(start, "%m/%d/%Y")
+        else:
+            start = self.first_opinion_date
+        if end:
+            end = datetime.strptime(end, "%m/%d/%Y")
+        else:
+            end = datetime.now() - timedelta(days=30)
+
+        self.back_scrape_iterable = [(start, end)]
```

Comment:
also does this make the iterable.  Looks like it just makes a list of the single tuple?   been a while since ive done a backscraper

Reply:
I updated the backscraper iterable creation so the list actually has more values. It would have been to many pages with a single query, making error recovery more costly. 

Say it failed  on page 20 of 100, we would have to make some manual calculation and restart things from the approximate date in page 20, which is not precise since default order is not by date, etc

So, it is indeed better to use properly the backscrape iterable!